src/main/java/frc/robot/commands/shooter/SpinFlywheels.java

```diff
@@ -35,6 +37,7 @@ public Aim(Shooter shooter, Vision eyes) {
 
   @Override
   public void initialize() {
+    initTime = Timer.getFPGATimestamp();
     if (m_type == null) {
```

Comment:
You should never use `null` like this. Make an enumeration constant called `AUTOMATIC` for this instead and include this as a part of the switch case. Though I will note, you shouldn't aim at the first thing the camera sees anyways... that's not how it was meant to be used. The camera estimates your position. You should be able to use that estimate to pivot your pose to the one you want.